R/lm_lin.R

```diff
@@ -65,7 +64,7 @@ lm_lin <- function(formula,
   cluster <- model_data$cluster
 
   # If Z is a factor, can't use variable name
-  # So get first column non intercept column
+  # So get first non-intercept column (always will be treatment)
```

Comment:
This is a reasonable assumption, make sure it's documented in the help page though - sometimes people get their factor levels out of order..

Reply:
We only accept something that has one variable on the rhs, but I guess you just made it clear that if that was a factor it could create multiple columns in the model matrix. Not sure how to handle this but I'll think about it.